### PR TITLE
(#44) IE issues for Drug App

### PR DESCRIPTION
**Pull request by karlikpj**
Closes (#44)

Fixes CSS for IE 11

### CODE REVIEW

**Review by seyilonge-nci — APPROVED** [review]

**Comment by alinai** [comment]
@karlikpj , On IE, the Search button is now aligned to the Search input box and looks good. The items in the table are still not aligned to the left and I could not verify the Patient Info button as this term does not have the button. Screen-shot is attached.

image

**Comment by alinai** [comment]
@karlikpj , The Patient Info button looks good on IE. The alignment issue on the table and the issue that the fonts are different on IE exists (Test term: bevacizumab)

File: src/components/molecules/definitionitem/DefinitionItem.scss [hunk 1]
```diff
@@ -30,7 +34,7 @@ a.dictionary-definiton__patient-information-button {
 	align-items: center;
 	font-family: Montserrat, Avant Garde, Arial, sans-serif;
 	font-size: 16px;
-	line-height: normal;
+	line-height: 29px;
```
Comment (arcepaul) [inline src/components/molecules/definitionitem/DefinitionItem.scss:37]: Line heights should be unitless wherever possible.  1.75 looks about right here.

**Review by arcepaul — APPROVED** [review]

**Comment by alinai** [comment]
The table alignment looks good on IE. Thanks @karlikpj  and @arcepaul !

**Pull request merged**